gsoft-inc/sg-orbit
# Jest tests for Menu & MenuTrigger

Issue: https://github.com/gsoft-inc/sg-orbit/issues/500 https://github.com/gsoft-inc/sg-orbit/issues/507 

## Summary

- Added Jest tests for Menu
- Added Jest tests for MenuTrigger
- Couple of fixes related to Menu & MenuTrigger
- Moved the `match` utility function in the disclosure component folder since it's used by this one

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes

- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
